assets/lokomotive-kubernetes/bootkube/variables.tf

```diff
@@ -94,7 +94,7 @@ variable "container_images" {
     calico_controllers      = "calico/kube-controllers:v3.13.3"
     hyperkube               = "k8s.gcr.io/hyperkube:v1.18.3"
     coredns                 = "coredns/coredns:coredns-"
-    pod_checkpointer        = "kinvolk/pod-checkpointer:83e25e5968391b9eb342042c435d1b3eeddb2be1"
+    pod_checkpointer        = "kinvolk/pod-checkpointer:c80b49365e2b0cebe3d6097a6ac78d1dc711699f"
```

Comment:
Should we push a new tag with the current commit on kinvolk-master: [d1c58443fe7d7d33aa5bf7d80d65d299be6e5847](https://github.com/kinvolk/bootkube/commit/d1c58443fe7d7d33aa5bf7d80d65d299be6e5847)

Reply:
done.